### ISSUE TITLE
Is this repository still maintained - any benefit to use it since Apollo allows local data management as well?

### ISSUE THREAD

**Issue by gregoryforel** [kamilkisiela/loona#539]

**Comment by ahmadalibaloch** [comment]
any update? It looks a very good start but why is this repo so silent and lonely?

**Comment by Rockson** [comment]
Yeah it's a good project imo, what lacks Apollo client right now is the ability to update multiple queries in a "reactive way". Today you have to do it imperatively. 

Apollo link watched mutation tries to solve the problem, but Apollo loona approach has a better developeer experience
https://github.com/haytko/apollo-link-watched-mutation

**Comment by Eraldo** [comment]
Is this library still maintained?

**Comment by twittwer** [comment]
@kamilkisiela Can you please say something about the status of the project 🙏
